package.json

```diff
@@ -13,25 +13,26 @@
   "author": "Sandro Munda",
   "license": "GPL-3.0",
   "dependencies": {
-    "bluebird": "^3.4.6",
-    "chalk": "^1.1.3",
-    "commander": "^2.9.0",
-    "dotenv": "^6.0.0",
+    "@oclif/plugin-warn-if-update-available": "1.7.0",
+    "bluebird": "3.5.1",
+    "chalk": "1.1.3",
+    "commander": "2.16.0",
+    "dotenv": "6.0.0",
     "expand-home-dir": "0.0.3",
-    "inquirer": "^6.2.0",
-    "jsonapi-serializer": "^3.4.1",
-    "jsonwebtoken": "^8.3.0",
-    "lodash": "^4.16.4",
-    "mkdirp": "^0.5.1",
-    "mongodb": "^3.1.8",
-    "mysql": "^2.11.1",
+    "inquirer": "6.2.1",
+    "jsonapi-serializer": "3.5.5",
+    "jsonwebtoken": "8.3.0",
+    "lodash": "4.17.10",
+    "mkdirp": "0.5.1",
+    "mongodb": "3.1.8",
+    "mysql": "2.16.0",
     "mysql2": "1.4.2",
-    "pg": "^6.1.0",
+    "pg": "6.4.2",
     "sequelize": "git://github.com/ForestAdmin/sequelize.git#feature/support-schema",
-    "superagent": "^3.8.3",
-    "superagent-promise": "^1.1.0",
-    "tedious": "^1.14.0",
-    "winston": "^2.2.0"
+    "superagent": "3.8.3",
+    "superagent-promise": "1.1.0",
+    "tedious": "1.15.0",
+    "winston": "2.4.3"
```

Comment:
Are you confident with all those changes?

Reply:
I looked at the existing package-lock and took the installed versions out of it, quite confident, I think it's worth the risk, the sooner the better